clouddq/log.py

```diff
@@ -64,14 +64,15 @@ def format(self, record):
         return super().format(record)
 
 
-def add_cloud_logging_handler(logger: Logger):
+def add_cloud_logging_handler(metadata, logger: Logger):
     client = google.cloud.logging.Client()
     handler = CloudLoggingHandler(
         client=client,
         name="clouddq",
         labels={
             "name": APP_NAME,
             "releaseId": APP_VERSION,
+            "metadata": str(metadata),
```

Comment:
Please ensure the metadata is not included as a string but as a nested object in the logs. I think this means instead of `str()` we need to use `json.dumps()`